src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/tests/Datagrid.test.js

```diff
@@ -39,20 +40,23 @@ jest.mock('../../../services/Translator', () => ({
 
 beforeEach(() => {
     datagridAdapterRegistry.has.mockReturnValue(true);
-    datagridAdapterRegistry.get.mockReturnValue(TableAdapter);
+    datagridAdapterRegistry.get.mockReturnValue({
+        Adapter: TableAdapter,
+        paginationType: 'default',
+    });
 });
 
 test('Change page in DatagridStore on pagination click', () => {
     const datagridStore = new DatagridStore('test');
-    const datagrid = shallow(<Datagrid views={['table']} store={datagridStore} />);
-    datagrid.find('Pagination').simulate('change', 3);
-    expect(datagridStore.setPage).toBeCalledWith(3);
+    const datagrid = mount(<Datagrid views={['table']} store={datagridStore} />);
```

Comment:
Because of the line `datagrid.find('Pagination').find('.next').simulate('click');`.
The `Pagination` is now inside the `PaginationDecorator` and won't get rendered using `shallow`